docs/comment_copy_guidelines.md

```diff
@@ -0,0 +1,127 @@
+  "as you know", "as you remember", "as you learned", "now that we all
+  undertstand x", because even though something was said, the person doesn't 
+  necessarily understood it.
+
+### Early exercises
+
+For the first exercises of a track, the following is _extra_ important:
+
+- **Name the difference** between what the person is doing, and what is
+  'idiomatic' in the language.
+- **Use the proper terms** and nomenclature, so people can recognize those
+  concepts elsewhere, and also can research them by themselves.
+- **Keep it relatively short**, avoiding a wall of text or overwhelming them
+  with advice. If they have a great experience in the first exercise, they'll
+  come back and you'll have many more opportunities to provide feedback on all
+  of the things that you noticed.
+- **Don't give the solution** as general advice for all Exercism mentoring. 
+  Learning sticks when people discover the answer for themselves. That's a 
+  tremendously exhilarating experience, and the emotional kick makes it 
+  memorable. However, if the discovery **doesn't** trigger a dopamine hit, it 
+  makes total sense to show what it looks like.
+- **Don't overly explain a concept**: don't go in depth about the underlying 
+  mechanisms of compilers and things. In this case it's more about this being 
+  one of the first exercises in the language track, and at this stage feedback
+  is more helpful if it's shorter and more directive.
+- **Give a link** that shows exactly how to do the concept, in a **tutorial** 
+  kind of way. Meaning: showing how to do things, not discussing why. This might
+  mean that the official docs of the language don't suffice, as these are often
+  a code reference and don't show you how to use it and how it works. _However_,
+  you should provide a link only for more in-depth exploration. The person
+  should be able to understand what you mean from the answer directly, without
+  following the link.
+
+### Examples
+
+In JavaScript, a student has written a top-level constant with `let`. 
+
+```markdown
+<!-- not following these guidelines -->
+As you know, everyone uses const, you shouldn't use let or var.
+```
+
+- The action is after the "explanation".
+- "As you know": we don't know if the student does.
+- "you shouldn't": you don't need the "you" to make this statement.
+- "everyone uses const": this isn't true ánd can make the student feel as if
+  they did something horribly wrong.
+- Lacks an actual explanation of _why_ the advice is given.
+
+```markdown
+<!-- better -->
+Prefer `const` and `let` over `var`. The `const` declaration stops a variable from being accidentally reassigned, which provides safety, and reduces cognitive load for someone reading the code. [This article](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75) explains the difference between the three.
+“this variable is never reassigned”, reducing cognitive load and improving 
```

Comment:
Is something missing here? This line reads a bit awkward to me.

Reply:
Suggestion didn't apply correctly.